plugins/woocommerce/includes/abstracts/abstract-wc-settings-api.php

```diff
@@ -950,20 +950,7 @@ public function validate_password_field( $key, $value ) {
 	 */
 	public function validate_textarea_field( $key, $value ) {
 		$value = is_null( $value ) ? '' : $value;
-		return wp_kses(
-			trim( stripslashes( $value ) ),
-			array_merge(
-				array(
-					'iframe' => array(
-						'src'   => true,
-						'style' => true,
-						'id'    => true,
-						'class' => true,
-					),
-				),
-				wp_kses_allowed_html( 'post' )
-			)
-		);
+		return wp_kses_post( trim( stripslashes( $value ) ) );
```

Comment:
Might be nice here again to note the significant change.

```php
 * @since {NEXT_VERSION} No longer allows storing IFRAME, which was allowed for "ShareThis",
 *                       but "ShareThis" is no longer part of Core.
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce/pull/48432/commits/e555e3b3d5914e9b8d42d147522b61b70f9c10fe